packages/cli/src/cmds/validator/signers/index.ts

```diff
@@ -91,7 +91,11 @@ export async function getSignersFromArgs(
         );
       },
     });
-    return await decryptKeystoreDefinitions(keystoreDefinitions, {...args, onDecrypt: needle});
+    return await decryptKeystoreDefinitions(keystoreDefinitions, {
```

Comment:
just minor thing I missed earlier, the `await` here should not be needed. Could consider adding the [no-return-await](https://eslint.org/docs/latest/rules/no-return-await) eslint rule to detect this

```suggestion
    return decryptKeystoreDefinitions(keystoreDefinitions, {
```